src/appshell/qml/ProjectPage/ProjectPage.qml

```diff
@@ -155,6 +155,15 @@ DockPage {
     panels: [
         DockPanel {
             id: tracksPanel
+            readonly property int effectsSectionWidth: 240
+            property bool showEffectsSection: false
+
+            onShowEffectsSectionChanged: {
+                const newWidth = root.verticalPanelDefaultWidth + (tracksPanel.showEffectsSection ? tracksPanel.effectsSectionWidth : 0)
+                tracksPanel.width = newWidth
```

Comment:
In qml there is a concept of binding: if you assign one property to another property (line 175), the value is automatically assigned to the first property when the second property changes. But if you later assign a fixed value to the first property, as in this line, then the binding stops working.

You of course need to remove the entire onShowEffectsSectionChanged block because you are already doing the same thing from line 175

Reply:
Is binding really what's happening line 175 in this case, though?
`root.verticalPanelDefaultWidth + (showEffectsSection ? effectsSectionWidth : 0)`

Reply:
Is binding really what's happening line 175 in this case, though?
`root.verticalPanelDefaultWidth + (showEffectsSection ? effectsSectionWidth : 0)` isn't a property but an expression using properties.
I would really have liked this to work, but I tried your suggestion and indeed, it doesn't.